lib/player.rb

```diff
@@ -6,6 +6,12 @@ def initialize()
     @balls = []
     @pins = []
   end
+# Method starts with empty arrays because we want to start fresh in between trurns.
+# Lines 18 - 23 > We are taking the argument "num_balls" and using the times method to get 16
```

Comment:
not necessarily 16 balls every time, it's actually num_balls * the 4 colors if that helps you think about it.

Reply:
I just put 16 because the argument is 4 but I changed my notes to reflect what you said. Thanks!